javaslang/src/test/java/javaslang/collection/IntMap.java

```diff
@@ -23,7 +23,7 @@
     @SuppressWarnings("unchecked")
     public static <T> IntMap<T> of(Map<Integer, T> original) {
         return original.isEmpty() ? (IntMap<T>) EMPTY
-                                  : new IntMap<>(original);
+                : new IntMap<>(original);
```

Comment:
It is my fault. We should provide a formatter that aligns rows correct when applied to a file. There most by some formatting option to do that. We have to fix the formatter and check it in (in a different PR). We can then also reformat the whole code base (modulo Generator.scala and generated code).

@ronanM has done all right. You may leave it as is!